### PR TITLE
Next

### PR DESCRIPTION
**Pull request by mstocke**

### CODE REVIEW

File: CHANGELOG.mkd [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # OpenXC iOS App Demo Changelog
 
-## v5.0.0
+## v6.0.0
+
+* Improvement: Support for Protobuf 3
+* Feature: Added disable trace loop and restart trace features
```
Comment (kranjanford) [inline CHANGELOG.mkd]: This feature is released in #v4.0.0

File: CHANGELOG.mkd [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+* Feature: Added base64 encoding to source ID for cloud export
```
Comment (kranjanford) [inline CHANGELOG.mkd]: Change this to" Feature:Upload Trace to URL"

File: CHANGELOG.mkd [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+* UI: Minor UI layout changes
+* Fix: Updated code signing data
+* Fix: Replaced numerous nil values with default values
```
Comment (kranjanford) [inline CHANGELOG.mkd]: change this to "UI:Dashboard signal missing values"

File: CHANGELOG.mkd [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+* Fix: Added AppTransportSecurity setting
```
Comment (kranjanford) [inline CHANGELOG.mkd]: Remove below two lines:
Fix: Added AppTransportSecurity setting.(This is part of Upload trace to URL)
Fix: Updated code signing data(our Provisioning profile expired it not a fix .Every one has their own developer certificate and profile .)

File: CHANGELOG.mkd (hunk 1 above)
Comment (mstocke) [reply CHANGELOG.mkd]: I updated the change log

File: CHANGELOG.mkd (hunk 4 above)
Comment (mstocke) [reply CHANGELOG.mkd]: I updated the change log

File: CHANGELOG.mkd (hunk 3 above)
Comment (mstocke) [reply CHANGELOG.mkd]: I updated the change log

File: CHANGELOG.mkd (hunk 2 above)
Comment (mstocke) [reply CHANGELOG.mkd]: I updated the change log

**Pull request merged**